docs/pages/branding/about.tsx

```diff
@@ -26,9 +26,9 @@ import GradientText from 'docs/src/components/typography/GradientText';
 import { getDesignTokens, getThemedComponents } from 'docs/src/modules/brandingTheme';
 import ROUTES from 'docs/src/route';
 import IconImage from 'docs/src/components/icon/IconImage';
-import SvgChat from 'docs/src/icons/SvgChat';
-import SvgPerson from 'docs/src/icons/SvgPerson';
-import SvgCard from 'docs/src/icons/SvgCard';
+import ForumRoundedIcon from '@material-ui/icons/ForumRounded';
+import PeopleRoundedIcon from '@material-ui/icons/PeopleRounded';
+import LocalAtmRoundedIcon from '@material-ui/icons/LocalAtmRounded';
```

Comment:
Using the Material Design icons is interesting. On one hand, the MD icons feel out of place in this design (and less appealing) but on the other, they are a reminder of our origin. In the future maybe we could use the opportunity to render the "Joy" icons.

Reply:
Yeah, you know... I'd still vote for using the custom icons 😬 But I think I lost the battle since they do add significant development complexity. Especially because I had one set for light and dark mode. But... who knows?! If we end up "natively" plugging in some other icon, we can come back and ditch these MD ones. They're mostly there for convenience and performance.

Reply:
For the light/dark icons duplication, I assume it could be fixed by leveraging opacity or duo-tones?
For convenience, I guess it's a tradeoff between going the extra mile to have something that has its own identity, or spending this time somewhere else. So a compromise to find with you and Jun 😁 
For the performance, AFAIK, the design of it has no/marginal impacts. What did you have in mind?

Reply:
What I meant by performance was having to maintain different svgs for dark and light mode in the repo, and then having to put in place functions to toggle between them at each instance, which could be too much. I'm not sure how much this weights exactly though. But you definitely gave an idea, to test with duo-tones. I'm going to check this out!